scripts/check_names.py

```diff
@@ -753,15 +783,18 @@ def parse_symbols(self):
             )
             my_environment = os.environ.copy()
             my_environment["CFLAGS"] = "-fno-asynchronous-unwind-tables"
```

Comment:
This is partially pre-existing (because this line was already there, but there's also a copied one below now) and also minor improvement, so it's up to you if it's worth to be done or not.
Since I wasn't aware of this flag and it took me some time to figure out what it does, the same might happen also to other people in the future, so I think a short comment on what this flag does would be nice. Wdyt?

Reply:
I think just to prevent issues down the line, we might want to define this flag before each build. It is slightly more verbose (even though duplicated) and prevents any issues with environment changes between the two function calls.

I know that in [C](https://www.evanjones.ca/setenv-is-not-thread-safe.html) at least the environment variable is not reliable between threads and potentially subprocesses :)